app/views/results/common/_peer_reviews.html.erb

```diff
@@ -12,15 +12,16 @@
       <% prs.each_with_index do |pr, i| %>
         <% if @current_role.student? %>
           <li> <%= link_to("#{PeerReview.model_name.human} #{i + 1}",
-                        view_marks_assignment_result_path(assignment.id, pr.result_id)) %> </li>
+                        view_marks_course_result_path(assignment.id, pr.result_id)) %> </li>
```

Comment:
This one is right. Please change the others to match (and double check the links work in the UI please)